GitHubTeams.ps1

```diff
@@ -319,131 +319,483 @@ filter Get-GitHubTeamMember
     return (Invoke-GHRestMethodMultipleResult @params | Add-GitHubUserAdditionalProperties)
 }
 
-function New-GitHubTeam
+filter Get-GitHubTeamProject
```

Comment:
From an API perspective, it _feels_ like it might make more sense to have `Get-GitHubProject` to have a parameterset that takes in the `TeamSlug` or `TeamName`  as opposed to having this _seperate_ command that gets Projects, but just those for teams.  My overall view here (which you'll see in the rest of the comments as well) is that you should look at the object that you're getting/removing/creating/changing, and determine if that's a `GitHub.Team`-specific object.  If it's not, then you're really interacting with _another_ object with a from a "Team perspective".